Watchman.AwsResources/Services/Sqs/QueueSource.cs

```diff
@@ -1,17 +1,13 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
 using Amazon.CloudWatch;
 using Amazon.CloudWatch.Model;
 
 namespace Watchman.AwsResources.Services.Sqs
 {
-    /// <summary>
-    /// read all queues that are "active"
-    /// i.e. have metrics in cloudwatch
-    /// See http://stackoverflow.com/a/40346581/5599
-    /// </summary>
     public class QueueSource : ResourceSourceBase<QueueData>
```

Comment:
This is also injected into `QueueNamePopulator` (old implementation) that presumably expects all queues (error or not) to be returned in one set. Is this going to break that? We could just copy the old implementation (with a new interface) to preserve that? as it's legacy anyway